composer.json

```diff
@@ -18,8 +18,8 @@
         "source": "https://github.com/cakephp/localized"
     },
     "require": {
-        "php": ">=5.4",
-        "cakephp/cakephp": "~3.1"
+        "php": ">=5.6",
+        "cakephp/cakephp": "^3.6"
```

Comment:
Is it possible to keep 3.5 compatibility here?

Reply:
Actually, it looks like setProvider was introduced in 3.4, so should i set it as min req?

Reply:
Yes, as low as possible :) But IMO 3.5 at this point would be fine, too.
Can you maybe also add the prefer lowest part as per https://www.dereuromark.de/2018/03/14/cakephp-3-6-is-coming/#prepare-your-plugins to make sure?